seacrowd/sea_datasets/sleukrith_ocr/sleukrith_ocr.py

```diff
@@ -0,0 +1,263 @@
+    isbn = {9781450353908},
+    publisher = {Association for Computing Machinery},
+    address = {New York, NY, USA},
+    url = {https://doi.org/10.1145/3151509.3151510},
+    doi = {10.1145/3151509.3151510},
+    abstract = {Analysis of ancient Khmer documents can be quite challenging due
+    to the elaborated shape of Khmer handwritten characters combined with the
+    complex structure of how words are formed from those characters. Palm leaf
+    manuscripts, one of the most well-known old Khmer documents, have been being
+    digitized and centralized; therefore, document analysis functions such as
+    text search capabilities are necessary but still remain unavailable for this
+    type of documents. In order to contribute to the progress of relevant
+    researches, we introduce in this paper a new dataset called SleukRith set
+    comprising of 657 pages of Khmer palm leaf manuscripts randomly selected
+    from various collections whose quality and digitization method are variable.
+    The dataset contains three types of data: isolated characters, words, and
+    lines. Each type of data is annotated with the ground truth information
+    which is very useful for evaluating and serving as a training set for common
+    document analysis tasks such as character/text recognition, word/line
+    segmentation, and word spotting. In order to serve as a base line, the
+    result of an evaluation study of Khmer isolated character recognition that
+    we have conducted on SleukRith Set using Convolutional Neural Network is
+    also presented.},
+    booktitle = {Proceedings of the 4th International Workshop on Historical Document Imaging and Processing},
+    pages = {1-6},
+    numpages = {6},
+    keywords = {palm leaf manuscript, ground truth, annotated dataset, Handwritten document analysis},
+    location = {Kyoto, Japan},
+    series = {HIP '17}
+}
+"""
+
+_DATASETNAME = "sleukrith_ocr"
+
+_DESCRIPTION = """\
+SleukRith Set is the first dataset specifically created for Khmer palm leaf
+manuscripts. The dataset consists of annotated data from 657 pages of digitized
+palm leaf manuscripts which are selected arbitrarily from a large collection of
+existing and also recently digitized images. The dataset contains three types of
+data: isolated characters, words, and lines. Each type of data is annotated with
+the ground truth information which is very useful for evaluating and serving as
+a training set for common document analysis tasks such as character/text
+recognition, word/line segmentation, and word spotting.
+"""
+
+_HOMEPAGE = "https://github.com/donavaly/SleukRith-Set"
+
+_LANGUAGES = ["khm"]
+
+_LICENSE = Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = {
+    "sleukrith-set": {
+        "images": "https://drive.google.com/uc?export=download&id=19JIxAjjXWuJ7mEyUl5-xRr2B8uOb-GKk",  # 1GB
+        "annotated-data": "https://drive.google.com/uc?export=download&id=1Xi5ucRUb1e9TUU-nv2rCUYv2ANVsXYDk",  # 11.7MB
+    },
```

Comment:
looks like this is unused anywhere on the dataloader, is it necessary>

Reply:
You are correct that it is unused here. My way of thinking is that if we consider the global variable to be the dataset metadata, it is okay to put it there. Inline comment also works in that case.

This `sleukrith-set` key is the raw data (the whole image), and `isolated-characters` is the processed data (cropped image per char).

Reply:
hmm prob an inline comment for clarifying this would be enough, like this one:

> This sleukrith-set key is the raw data (the whole image), and isolated-characters is the processed data (cropped image per char).

And it's okay if we put it in the `_URL` and note down the ones we're using is `isolated-characters` key instead